src/pages/career.tsx

```diff
@@ -44,6 +49,12 @@ export default Career;
 
 export const CareerPageQuery = graphql`
   query ($language: String!) {
+    hero: file(relativePath: { eq: "office/sy-office-01.jpg" }) {
+      childImageSharp {
+        gatsbyImageData(layout: FULL_WIDTH)
```

Comment:
`FULL_WIDTH` is the only thing we need. The default props include a set of breakpoints which means we get the image in different srcSets while being set to display as 100%.